modules/structs/repo_actions.go

```diff
@@ -32,3 +32,36 @@ type ActionTaskResponse struct {
 	Entries    []*ActionTask `json:"workflow_runs"`
 	TotalCount int64         `json:"total_count"`
 }
+
+// CreateActionWorkflowDispatch represents the payload for triggering a workflow dispatch event
+// swagger:model
+type CreateActionWorkflowDispatch struct {
+	// required: true
+	// example: refs/heads/main
+	Ref string `json:"ref" binding:"Required"`
+	// required: false
+	Inputs map[string]any `json:"inputs,omitempty"`
```

Comment:
Like you mentioned, my initial PR from my side used the following instead

```suggestion
	Inputs map[string]string `json:"inputs,omitempty"`
```

However I don't really care if you provide a complex json object in the api call, even if this is not valid in GitHub Actions api.

Followup of https://github.com/go-gitea/gitea/pull/33545#discussion_r1949308936

Reply:
Done in 855103d9212558890ae676d2ee3d30b2b4984834